cmd/logcli/client.go

```diff
@@ -55,7 +55,9 @@ func listLabelValues(name string) (*logproto.LabelResponse, error) {
 
 func doRequest(path string, out interface{}) error {
 	url := *addr + path
-	log.Print(url)
+	if !*quiet {
```

Comment:
@sh0rez What do you think about using log level or something instead of putting this condition at all the places where a print statement is used?

Reply:
We are using package `log` here... Which does not support log levels. When using this, I believe the `if verbose` are considered kinda idiomatic. I'm fine with this.

Reply:
I have no problem writing these and I like the simplicity of staying with the standard library